store/postgres/src/relational_queries.rs

```diff
@@ -932,7 +967,114 @@ impl<'a> QueryFilter<'a> {
         QueryFilter {
             filter,
             table: self.table,
+            layout: self.layout,
+            block: self.block,
+            table_prefix: self.table_prefix.clone(),
+        }
+    }
+
+    fn child(
+        &self,
+        attribute: &Attribute,
+        entity_type: &'a EntityType,
+        filter: &'a EntityFilter,
+        derivative: &'a EntityFilterDerivative,
+        mut out: AstPass<Pg>,
+    ) -> QueryResult<()> {
+        let child_table = self
+            .layout
+            .table_for_entity(entity_type)
+            .expect("Table for child entity not found");
+
+        let child_prefix = "i.";
+        let parent_prefix = "c.";
+
+        out.push_sql("exists (select 1");
+
+        out.push_sql(" from ");
```

Comment:
Very minor but these two `push_sql` could be collapsed into one

Reply:
a7c70e42cbb1919be46b9af02c4a129f228217b1